AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet1Solutions.lean

```diff
@@ -0,0 +1,78 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-
+Welcome!
+
+This is an interactive problem set, where you'll learn to use Lean 4,
+a theorem prover and programming language, while exploring the basics
+of abstract algebra. Through exercises on groups, rings, and fields, you'll
+formalize and verify mathematical proofs. By the end, you should be comfortable
+with using Lean 4 to prove and verify theorems.
+
+This problem set is written under the assumption that you, the user, have
+already completed the Natural Number Game. Additionally, it is assumed that
+the user is in the process of taking an abstract algebra class (i.e. 401 or 501),
+so explanations of concepts will be more concise than a textbook's.
+
+There are solutions provided for your reference. Should you be interested in
+learning more about specific tactics, Lean 4 documentation is available online.
```

Comment:
Links to the documentation would be useful

Reply:
Put in a couple links; wasn't sure if they were the best resources. If there are better ones to use, please suggest!